src/newscale/__init__.py

```diff
@@ -1,3 +1,3 @@
 """Simple package to demo project structure.
 """
-__version__ = "0.0.1"
+__version__ = "0.1"
```

Comment:
fixed!

Reply:
re: pdfs, I like the no-binaries thing in principle, but these were kinda hard to find.
Generally, I'd just link to them, but since these were quite difficult to find on a link that might change, I want to keep them here. (And if you really want to apply this to the extreme, should we be revoking the PDFs of schematics on other projects?)

I really do think datasheets for a python-package that is a driver for the datasheet component makes sense here. If you have a location in mind where we can store these datasheets, I'm ok to remove them, but until then, having them here makes them easy to find for others.